pkg/platform/aks/aks.go

```diff
@@ -71,6 +73,9 @@ type config struct {
 	WorkerPools []workerPool `hcl:"worker_pool,block"`
 
 	KubernetesVersion string
+
+	// TODO: Transient change - remove when refactoring platform interface.
+	Backend *backend.Backend
```

Comment:
This is a noticeable change to the UI, since the block is being moved inside `cluster {}` block.